### DISCUSSION TITLE
How to get real client ip

### DISCUSSION THREAD

**Discussion by fulljackz** [firefly-iii/firefly-iii#8721] [General]
Hi,

I'm using firefly with docker compose in a swarm setup with a global traefik frontend.
Everything is ok but in the logs I only have the traefik source ip instead of my client ip.

Is there any way to configure the correct headers to have the real client ip ?

For information, everything is already configured on the traefik side (others stacks have the real client ip).

Regards,

**Comment by JC5** [comment]
Idk, I'm not sure. Perhaps x-forwarded-for?

**Reply by fulljackz** [reply]
Yep, absolutely or maybe just enable mod_remoteip : https://httpd.apache.org/docs/2.4/fr/mod/mod_remoteip.html ?
